apps-rendering/src/client/newsletterEmbeds.ts

```diff
@@ -0,0 +1,61 @@
+type NewsletterHeightEventType = { source: MessageEventSource; origin: string };
+type MessageType = {
+	type: string;
+	value: string;
+};
+
+const allowedOrigins = ['https://www.theguardian.com'];
+export default (): Promise<void> => {
+	const allIframes: HTMLIFrameElement[] = [].slice.call(
+		document.querySelectorAll('.email-sub__iframe'),
+	);
+
+	// Tell the iframes to resize once this script is loaded
+	// Otherwise, earlier resize events might be missed
+	// So we don't have to load this script as a priority on each load
+	allIframes.forEach((iframe) => {
+		iframe.contentWindow?.postMessage('resize', '*');
```

Comment:
Is there anything that prevents us using `https://www.theguardian.com` as the `targetOrigin` here instead of `*`?

Reply:
Oh, yes! This message should only be sent to frames on `https://www.theguardian.com`. It makes testing on `CODE` a little harder but I think the tradeoff is worth it